common/src/main/java/com/alibaba/nacos/common/http/client/NacosRestTemplate.java

```diff
@@ -336,6 +460,16 @@ public NacosRestTemplate(HttpClientRequest requestClient) {
         }
     }
     
+    private HttpClientRequest requestClient() {
+        if (CollectionUtils.isNotEmpty(interceptors)) {
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Execute via interceptor");
```

Comment:
Should we print the interceptor name or class name?

Reply:
okay